src/pages/Aggregator/Aggregator.component.tsx

```diff
@@ -94,6 +108,11 @@ export const AggregatorComponent = ({
     }
   }, [amount, getReceiveAmount, setValue]);
 
+  useEffect(() => {
+    resetField(AggregatorInputs.SendAmount);
+    resetField(AggregatorInputs.ReceiveAmount);
+  }, [flowState, resetField, setValue]);
```

Comment:
Now `setValue` can be removed from the dependency list

```suggestion
  }, [flowState, resetField]);
```